<launch>
  <node name="music_player" type="music_player.py" pkg="dream_machine"/>
  <node name="voice_handler" type="voice_handler.py" pkg="dream_machine"/>
</launch>
